src/components/molecules/MoleculeArticleCard.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="mdc-list-item__start">
+    <div class="mdc-list-item__graphic">
+      <AtomIcon :src="src" />
+    </div>
+    <div class="mdc-list-item__content">
+      <h1 class="mdc-list-item__primary-text">{{ name }} ({{ amount }})</h1>
+      <p class="mdc-list-item__secondary-text">Läuft ab in {{ expiringIn }} Tagen || MHD: {{ mhd }}</p>
+    </div>
+    <div class="mdc-list-item__end">
+      <AtomMaterialIcon class="ateButton">restaurant</AtomMaterialIcon>
+      <AtomMaterialIcon class="deleteButton">delete</AtomMaterialIcon>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import AtomIcon from '@/components/atoms/AtomIcon.vue';
+import AtomMaterialIcon from '@/components/atoms/AtomMaterialIcon.vue';
+
+defineProps({
+  name: {
+    type: String,
+    required: true,
+  },
+  src: {
+    type: String,
+    required: true,
+  },
+  mhd: {
+    type: String,
+    required: true,
+  },
+  expiringIn: {
+    type: String,
+    required: true,
+  },
+  amount: {
+    type: String,
+    required: true,
+  },
+}
+);
```

Comment:
Code-Styling: remove line break 
```suggestion
});
```